2/수박수박수박수박수?.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  let answer = "";
+  for (let i = 1; i <= n; i++) {
```

Comment:
```suggestion
  let answer = "";
  
  for (let i = 1; i <= n; i++) {
```
변수 선언과 구문 사이 줄바꿈 해줍니다.